Testscripts/Windows/DM-HOTADD-STRESS-NG.ps1

```diff
@@ -141,36 +137,42 @@ function Main {
            Throw "Unable to start job for creating pressure on $vm1Name" | Tee-Object -Append -file $summaryLog
         }
         # Wait for stress-ng to start and the memory assigned/demand gets updated
-        Start-Sleep -Seconds $sleepTime
-        [int64]$vm1Demand = ($vmInfo.MemoryDemand/1MB)
-        # Get memory stats for vm1 after stress-ng starts
-        [int64]$vm1Assigned = ($vmInfo.MemoryAssigned/1MB)
+        for ($i = 1; $i -lt 20; $i++) {
```

Comment:
what do you choose '20' for for-loop? Only 20 iterations?

Reply:
the worst situation is to sleep 20 * 3 seconds, wait for the memory change after stress-ng start running, once the current memory increase, exit this loop, before after sleep 20 seconds, the memory already increase and decease, so case failed in a high rate.